Pcap++/header/DpdkDevice.h

```diff
@@ -804,6 +810,7 @@ namespace pcpp
 		int m_Id;
 		MacAddress m_MacAddress;
 		uint16_t m_DeviceMtu;
+		int32_t m_MbufDataSize;
```

Comment:
maybe this can be of size `int16_t`?
The size of `data_room_size` is `uint16_t`: https://doc.dpdk.org/api/rte__mbuf_8h.html#a593921f13307803b94bbb4e0932db962